changelog.d/7277.wip

```diff
@@ -0,0 +1 @@
+[Device Management] Show correct device type icons
```

Comment:
Should this be 'wip'?

Reply:
I have used ".wip" for all related PRs for the new device manager feature since it is under development (behind feature flag).